contents/javascript/staticTypingWithTypescript.md

```diff
@@ -0,0 +1,54 @@
+<frontmatter>
+  title: Static Typing with TypeScript
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Static Typing with TypeScript
+
+**Author(s): [Lu Yang Kenneth](https://github.com/luyangkenneth)**
+
+Reviewer(s): WIP FILL THIS OUT LATER
+
+
+## What is Static Typing?
+
+**Statically typed languages** like Java, Go, and C++ are able to catch type-related errors at compile time, while in **dynamically typed languages** like JavaScript, such errors are not as easily discoverable because the types of variables are only known at runtime.
```

Comment:
I feel like bolding these phrases don't really add much impact. You might want to consider adding tooltips/popovers to explain what exactly statically or dynamically typed languages are. Perhaps having some code snippets from a statically typed language vs dynamically typed language would help to highlight the difference.

Reply:
I think it's a reasonable assumption that readers know what statically / dynamically typed languages are, so I meant for this part to just set the stage for the rest of the introduction. I don't have the intention to go into too much explanation here.

The bolding wasn't meant to have any impact actually haha. It was added as an afterthought because I thought the page might look more aesthetic with some bolded words 😛